var/spack/repos/builtin/packages/py-onnx/package.py

```diff
@@ -48,12 +58,18 @@ class PyOnnx(PythonPackage):
     depends_on("py-protobuf@3.12.2:3.20.1", type=("build", "run"), when="@1.12.0")
     depends_on("protobuf@3.20.2:3", when="@1.13.0:")
     depends_on("py-protobuf@3.20.2:3", type=("build", "run"), when="@1.13.0:")
```

Comment:
Is this still true? Seems like protobuf 4 is supported now: https://github.com/onnx/onnx/pull/4956

Reply:
Feel free to modify this list in whatever way is easiest to maintain in terms or sorting or version ranges. It's pretty complex at the moment.